### DISCUSSION TITLE
Is that possible to build jar with dependencies using sbt ?

### DISCUSSION THREAD

**Discussion by andrey-shishkin** [sbt/sbt#7544] [General]
Hi there.
Couldn't find the answer in the internet, so I'm asking it here.
Most of the posts suggest to use `assembly plugin`, but I'm curious is there a way to build jar with dependencies using sbt to be able to run it using `java -jar` command ?
I've tried to use `sbt package`, but it builds thin jar which is impossible to run using `java -jar` command

**Comment by adpi2** [comment]
The answer is in the question. You need to use the `assemby` task of the `sbt-assembly` plugin: https://github.com/sbt/sbt-assembly#assembly-task

**Reply by andrey-shishkin** [reply]
Thank you!
